en/drafts/originals/image-classification-neural-networks.md

```diff
@@ -25,150 +25,154 @@ doi: tbc
+Hopefully, the choice of the tools in this tutorial will allow you to focus on the broader concepts surrounding neural networks without worrying as much about coding. It's worth mentioning, however, that Python and R are vastly more popular at the production level, and much of the cutting-edge work in machine learning relies on these two languages rather than the toolset shown in this tutorial. If you are interested in expanding your knowledge of neural networks, see *Programming Historian's* excellent articles on [Computer Vision for the Humanities](/en/lessons/computer-vision-deep-learning-pt1) and [Interrogating a National Narrative with Recurrent Neural Networks](/en/lessons/interrogating-national-narrative-gpt).
 
 # Setup and Dataset
 
-To begin, create a new folder called `projects`. This folder will hold all relevant files and images. To train the neural network in [Google's Teachable Machine](https://teachablemachine.withgoogle.com/), you will need a collection of labeled images since most neural networks are geared towards supervised learning. Machine learning can be divided into two forms: supervised and unsupervised learning. Unsupervised learning looks for patterns inside of data and tries to group data that is alike together. You may have seen the use of some unsupervised machine learning algorithms such as [K-Means Clustering](https://en.wikipedia.org/wiki/K-means_clustering) and [Latent Dirichlet Allocation](https://en.wikipedia.org/wiki/Latent_Dirichlet_allocation) in digital humanities research. Supervised learning, on the other hand, looks at data that is already labeled.
+To begin, create a new folder called `projects`. This folder will hold all relevant files and images. To train the neural network in [Google's Teachable Machine](https://teachablemachine.withgoogle.com/), you will need a collection of labeled images as most neural networks are geared towards what is known as 'supervised learning'. Machine learning can be divided into two forms: supervised and unsupervised learning. Supervised learning involves data that is already labeled. Unsupervised learning, on the other hand, involves identifying patterns and grouping data that is alike together. You may have seen the use of some unsupervised machine learning algorithms such as [K-Means Clustering](https://perma.cc/39XM-PBAS) and [Latent Dirichlet Allocation](https://perma.cc/JM3P-BZWN) in digital humanities research. 
 
 For this tutorial, we will download a dataset of paintings from [ArtUK](https://artuk.org/), which provides access to works that meet the UK's requirements for "public ownership." Before its launch, most of the UK's paintings in public ownership were not in regular circulation. ArtUK combats this by providing the general public access to these materials.
```

Comment:
You write that before the launch of ArtUK "most of the UK's paintings in public ownership were not in regular circulation".  I don't think this is true ! For example, the extensive paintings collections of the National Gallery, Tate and the V&A (among others) have long been in circulation. Providing the general public with access to their collections has always been fundamental to the very existence of these institutions !

I'm also confused by the phrasing: "works that meet the UK's requirements for 'public ownership'". 
Is this a quote  ?  What is meant by this ?